### PR TITLE
Improved grammar and spelling in chapters 'Type Classes', 'Omega' and 'Micromega' of the Reference Manual.

### PR DESCRIPTION
**Pull request by wkolowski**
**Kind:** documentation

Mostly spelling and grammar. I also made the grammar for micromega formulas more readable (at least in my opinion).

Two issues:
1. I think that the syntax for the command "Instance" in the chapter "Type Classes" looks quite bad.
2. In the chapter "Omega", there's this quote:
"Note that each inequation defines a half space in the space of real value of the variables"

What exactly is meant here? Is "real value of the variables" some technical term or just a bad translation from French?

### CODE REVIEW

**Comment by Zimmi48** [comment]
> I also made the grammar for micromega formulas more readable (at least in my opinion).

Indeed. And it didn't used to be this bad (https://coq.inria.fr/distrib/V8.7.2/refman/micromega.html). Yet another problem coming from the Sphinx migration!

> What exactly is meant here? Is "real value of the variables" some technical term or just a bad translation from French?

I would say just a bad wording altogether. It wouldn't sound better in French. I think it is just talking about the underlying vector space (where you don't care if variables are integer or not).

**Comment by Zeimer** [comment]
I replaced it with "Note that each inequation cuts the Euclidean space in half."

**Comment by Zimmi48** [comment]
Much better, thanks!

**Review by Zimmi48 — APPROVED** [review]

File: doc/sphinx/addendum/omega.rst [hunk 1]
```diff
@@ -8,15 +8,12 @@ Omega: a solver for quantifier-free problems in Presburger Arithmetic
...
 
-:tacn:`omega` solves a goal in Presburger arithmetic, i.e. a universally
-quantified formula made of equations and inequations. Equations may
-be specified either on the type ``nat`` of natural numbers or on
-the type ``Z`` of binary-encoded integer numbers. Formulas on
-``nat`` are automatically injected into ``Z``.  The procedure
+:tacn:`omega` is a tactic for solving goals in Presburger arithmetic,
+i.e. for proving formulas made of equations and inequations over the
+type ``nat`` of natural numbers or the type ``Z`` of binary-encoded integers.
+Formulas on ``nat`` are automatically injected into ``Z``. The procedure
```
[the remark this answers is not in the document]
Comment (Zeimer) [reply doc/sphinx/addendum/omega.rst]: Done.

**Pull request merged**